### PR TITLE
feat(content): Smart watches can sense wind, addition of "WINDMETER" flag.

### PR DESCRIPTION
**Pull request by Lamandus**
## Purpose of change
There was only one Object that could sense the wind, that was the "Doppler Radar Turbo 2000". Cringy name, I know. And it dropped only by zombie scientist by a very small change. Instead of relying on that thing, we now have the flag "WINDMETER" that can print out weather information (as barometer, thermometer and gyrometer). 

Already you can buy Wind meter accessories for your phone, so why shouldn't we have it build in into your smartphone. https://mrtarget.de/skywatch-windoo-2-wind-meter-windmesser-anemometer-fuer-das-smartphone (not sponsored)

## Describe the solution
Add Windmeter as a flag. This is only used by the Turbo thing and the Smart Watch for now. Can be expanded to other object in the future.

## Testing
spawned a smart watch and let it happen

## Additional context
grafik

### CODE REVIEW

**Comment by Lamandus** [comment]
if it gets merged, @chaosvolt can you be so kind and post about it in the json-changes discord? thx.

File: src/iuse.cpp [hunk 1]
```diff
@@ -9024,7 +9024,7 @@ int iuse::weather_tool( player *p, item *it, bool, const tripoint & )
         }
     }
 
-    if( it->typeId() == itype_weather_reader ) {
+    if( it->has_flag( flag_WINDMETER ) ) {
```
Comment (scarf005) [inline src/iuse.cpp:9027]: is the `itype_weather_radar` removed?

Comment (Lamandus) [reply src/iuse.cpp:9027]: weather reader. Yes, it got removed, but it got the flag.

Comment (Lamandus) [reply src/iuse.cpp:9027]: > is the `itype_weather_radar` variable declaration removed?

the declaration is getting used in other places, too.

**Review by scarf005 — APPROVED** [review]

**Pull request merged**